src/client/mapblock_mesh.cpp

```diff
@@ -49,7 +49,7 @@ void MeshMakeData::fillBlockDataBegin(const v3s16 &blockpos)
 
 	m_vmanip.clear();
 	VoxelArea voxel_area(blockpos_nodes - v3s16(1,1,1) * MAP_BLOCKSIZE,
-			blockpos_nodes + v3s16(1,1,1) * MAP_BLOCKSIZE*2-v3s16(1,1,1));
+			blockpos_nodes + v3s16(1,1,1) * MAP_BLOCKSIZE*3-v3s16(1,1,1));
```

Comment:
shouldn't this use side_length or something?

Reply:
Fixed now